templates/shared/user/profile_big_avatar.tmpl

```diff
@@ -17,9 +17,20 @@
 		{{if .EnableFeed}}
 			<a href="{{.ContextUser.HomeLink}}.rss"><i class="ui text grey gt-ml-3" data-tooltip-content="{{.locale.Tr "rss_feed"}}">{{svg "octicon-rss" 18}}</i></a>
 		{{end}}
-		<div class="gt-mt-3">
-			<a class="muted" href="{{.ContextUser.HomeLink}}?tab=followers">{{svg "octicon-person" 18 "gt-mr-2"}}{{.NumFollowers}} {{.locale.Tr "user.followers"}}</a> · <a class="muted" href="{{.ContextUser.HomeLink}}?tab=following">{{.NumFollowing}} {{.locale.Tr "user.following"}}</a>
-		</div>
+		{{if or .NumFollowers .NumFollowing}}
+			<div class="gt-mt-3">
+				{{/* The conditions here make the template a little bit more bloated, but more "looked after". The hardcoded 0 is a micro-optimization, as we have already checked that the value is 0 anyways. */}}
+				{{if and .NumFollowers (not .NumFollowing)}}
+					{{svg "octicon-person" 18 "gt-mr-2"}}<a class="muted" href="{{.ContextUser.HomeLink}}?tab=followers">{{.NumFollowers}} {{.locale.Tr "user.followers"}}</a> · <span>0 {{.locale.Tr "user.following"}}</span>
```

Comment:
Make icon clickable by putting it into `<a>` like before?

Reply:
I was not sure if that made sense, because the behavior of clicking on that icon would change seemingly arbitrarily. It's not like the other icons are clickable.

On the other hand, I am not sure if making an option (obviously the first one) in some edge cases *harder* to click simply because the second option is harder to click is preferable.

Reply:
I guess it's fine as-is then. The fact that only the first value has a icon makes it seem inconsistent because the icon is obviously meant for both.

Reply:
Fine as-is, as in the way that I did it, or the way that it currently is in Gitea?

Reply:
Fine without the icon clickable.